skins/planetmemes.scss

```
@@ -0,0 +1,10 @@
+.trustwallet {
```

The css selector should similar to the skin file name. `skin.scss` `.skin {}`.
```suggestion
.planetmemes {
```